lib/process-services/src/lib/task-list/components/task-form/task-form.component.ts

```diff
@@ -117,6 +117,14 @@ export class TaskFormComponent implements OnInit {
   @Output()
   cancel = new EventEmitter<void>();
 
+  /** Emitted when the task is claimed. */
+  @Output()
+  claim: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
not sure this logic should be in the form

Reply:
we have the same on the task form cloud. The name of the event should be changed. @eromano https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/process-services-cloud/src/lib/task/task-form/components/task-form-cloud.component.ts#L78

Reply:
Let's call them as the one in the cloud